### PR TITLE
Update open-window.csl

### PR DESCRIPTION
**Pull request by Griever158**
The previous style was created by an ex-employee on their own. This proposed version aligns correctly with the 2022 Open Window Referencing Style, as confirmed by myself, Karl van Heerden, the subject manager for Visual Culture and member of the referencing team for the Open Window. For any queries, please feel free to contact me at karl@openwindow.co.za

### CODE REVIEW

File: open-window.csl [hunk 1]
```diff
@@ -177,38 +146,104 @@
...
+    </choose>
+  </macro>
+  <macro name="author-count">
+    <names variable="author">
+      <name form="count"/>
+    </names>
+  </macro>
   <citation disambiguate-add-year-suffix="true" year-suffix-delimiter="," disambiguate-add-names="true" disambiguate-add-givenname="false" collapse="year-suffix">
     <sort>
       <key macro="date-issued-no-parentheses"/>
```
Comment (adam3smith) [inline open-window.csl:234]: The sorting here is incorrect, isn't it? Style guide says to sort alphabetically (7.1)

File: open-window.csl [hunk 2]
```diff
@@ -177,38 +146,104 @@
...
       <key macro="author"/>
     </sort>
     <layout prefix="(" suffix=")" delimiter="; ">
```
Comment (adam3smith) [inline open-window.csl:237]: And the delimiter here should be a comma I think

File: open-window.csl [hunk 3]
```diff
@@ -260,36 +293,44 @@
...
             <text macro="title"/>
             <text variable="genre"/>
-            <text variable="publisher"/>
+            <text macro="publisher"/>
           </else-if>
-          <else-if type="chapter paper-conference entry-encyclopedia entry-dictionary" match="any">
+          <else-if type="chapter entry-encyclopedia entry-dictionary" match="any">
             <text macro="title"/>
             <group delimiter=" ">
               <text term="in" text-case="capitalize-first"/>
```
Comment (adam3smith) [inline open-window.csl:325]: Book chapters look wrong -- should be lowercase in, lowercase edited by and commas between elements, right?

**Review by adam3smith — CHANGES_REQUESTED** [review]
Sorry for the delay in reviewing. A couple of things aren't quite right yet, best I can tell -- could you please check?

File: open-window.csl (hunk 1 above)
Comment (Griever158) [reply open-window.csl]: Hi Adam! Thank you so much for getting back to me! You are correct, the ordering of citations should be author then year. I have made this change, you should see it already?

File: open-window.csl (hunk 2 above)
Comment (Griever158) [reply open-window.csl:237]: The delimiter here should be a semicolon, this is correct.

File: open-window.csl (hunk 3 above)
Comment (Griever158) [reply open-window.csl:325]: So, funny story. They were wrong, but last week we revised our style guide, and we've actually changed it to be consistent with how it is formatted here: _Title_. Edited by J Smith. It's more consistent with translated works and interviews now. So this format is now correct.

**Comment by adam3smith** [comment]
Thanks!

**Pull request merged**